README.md

```diff
@@ -54,7 +54,14 @@ The endpoints provided by this microservice should now be available through Cont
 
 1 - Create and complete your `dev.env` file with your configuration. The meaning of the variables is available in this [section](#configuration-environment-variables). You can find an example `dev.env.sample` file in the project root.
 
-2 - Execute the following command to run Control tower:
+2 - Create a docker network for services including elasticsearch and rabbitmq that are shared across document
+microservices:
+
+```
+docker network document-adapter
+```
+
+3 - Execute the following command to run Control tower:
```

Comment:
if you are reviewing the docs (which is something I would encourage, but perhaps as part of a separate PR) you should remove references to CT, which we are no longer using.

Reply:
Yes, I do read and reread the docs and appreciate the detail but this line isn't a new addition to this doc. As it stands, CT is required for dev and I'll defer removing these references to when there is a working and documented alternative.